src/main/java/uk/gov/laa/ccms/caab/mapper/ApplicationMapper.java

```diff
@@ -0,0 +1,645 @@
+  @Mapping(target = "assessedIncome", ignore = true)
+  @Mapping(target = "assessedIncomeFrequency", ignore = true)
+  @Mapping(target = "assessmentDate", ignore = true)
+  @Mapping(target = "title", ignore = true)
+  @Mapping(target = "surname", ignore = true)
+  @Mapping(target = "publicFundingApplied", ignore = true)
+  @Mapping(target = "partyId", ignore = true)
+  @Mapping(target = "nationalInsuranceNumber", ignore = true)
+  @Mapping(target = "middleNames", ignore = true)
+  @Mapping(target = "legalAided", ignore = true)
+  @Mapping(target = "firstName", ignore = true)
+  @Mapping(target = "employmentStatus", ignore = true)
+  @Mapping(target = "employmentAddress", ignore = true)
+  @Mapping(target = "employerName", ignore = true)
+  @Mapping(target = "displayName", ignore = true)
+  @Mapping(target = "displayAddress", ignore = true)
+  @Mapping(target = "dateOfBirth", ignore = true)
+  @Mapping(target = "courtOrderedMeansAssessment", ignore = true)
+  @Mapping(target = "confirmed", ignore = true)
+  @Mapping(target = "certificateNumber", ignore = true)
+  @Mapping(target = "award", ignore = true)
+  @Mapping(target = "auditTrail", ignore = true)
+  Opponent toOrganisationOpponent(OtherParty otherParty);
+
+  @Mapping(target = "lscCaseReference", source = "caseReferenceNumber")
+  @Mapping(target = "clientReference", source = "client.clientReferenceNumber")
+  @Mapping(target = "clientFirstName", source = "client.firstName")
+  @Mapping(target = "clientSurname", source = "client.surname")
+  @Mapping(target = "categoryOfLaw", source = "categoryOfLawDesc")
+  @Mapping(target = "providerCaseReference", source = "providerReferenceNumber")
+  @Mapping(target = "feeEarner", source = "feeEarnerName")
+  @Mapping(target = "status", source = "caseStatus")
+  @Mapping(target = "relationToCase", source = "linkType")
+  LinkedCase toLinkedCase(uk.gov.laa.ccms.soa.gateway.model.LinkedCase soaLinkedCase);
+
+  @Mapping(target = "status", source = "soaPriorAuthority.decisionStatus")
+  @Mapping(target = "summary", source = "soaPriorAuthority.description")
+  @Mapping(target = "type", source = "priorAuthorityTypeLookup")
+  @Mapping(target = "justification", source = "soaPriorAuthority.reasonForRequest")
+  @Mapping(target = "amountRequested", source = "soaPriorAuthority.requestAmount")
+  @Mapping(target = "valueRequired", source = "priorAuthorityTypeLookup.valueRequired")
+  @Mapping(target = "ebsId", ignore = true)
+  @Mapping(target = "auditTrail", ignore = true)
+  PriorAuthority toPriorAuthority(PriorAuthorityMappingContext priorAuthorityMappingContext);
+
+  @Mapping(target = "code.id", source = "key.code")
+  @Mapping(target = "code.displayValue", source = "key.description")
+  @Mapping(target = "type", source = "key.dataType")
+  @Mapping(target = "lovLookUp", source = "key.lovCode")
+  @Mapping(target = "value", source = "value")
+  //  @Mapping(target = "mandatory", source = "mandatoryFlag")
```

Comment:
need to remove this?

Reply:
No - there's a bug CCLS-1895 to add the mandatory flag to PriorAuthorityDetail. I've pulled it into the sprint.

Reply:
mandatory flag now mapped correctly